tests/main/interfaces-ssh-control/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Check that SSH configuration accessible through an interface
+
+details: |
+    This test makes sure that a snap using the ssh-control interface can
+    access the system ssh configuration.
+
+systems: [ubuntu-core-16-64]
```

Comment:
Could you please add the `ubuntu-core-16-arm-32` and `ubuntu-core-16-arm-64` systems here? This way the test will be executed on the pi2, pi3 and dragonboard too.